src/msal-client-credentials-flow/README.md

```diff
@@ -0,0 +1,163 @@
+| **Client secret**                 | _**Value** of the client secret (not its ID)_                                | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).        |
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Configure the web API
+
+1. Open the _~/msal-client-credentials-flow/appsettings.json_ file in your code editor and modify the following values values with those from your [app's registration in the Azure portal](https://docs.microsoft.com/azure/active-directory/develop/quickstart-configure-app-expose-web-apis):
+
+   ```json
+   "ClientId": "Enter_the_Application_Id_here",
+   "TenantId": "Enter_the_Tenant_Info_Here",
+   "ClientSecret": "Enter_the_Application_CLient_Secret_Here"
+   ...
+   "RelativePath": "Enter_the_Application_Object_Id_Here",
+   ```
+
+## Run the application
+
+### 1. Run the web API
+
+1. Execute the following command to get the app up and running:
+
+   ```bash
+   dotnet run
+   ```
+
+### 2. Send request to the web API
+
+1. Once the app is listening, execute the following to send the a request.
+
+   ```bash
+   curl -X GET https://localhost:5001/application -ki
+   ```
+
+   If everything worked, you should receive a response from the downstream web API (Microsoft Graph, in this case) similar to this:
+
+   ```console
+   $ curl https://localhost:5001/api/application -ki
+   {
+      "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#applications/$entity",
+      "id": "537a552c-58b7-4468-abdf-a7cbfa000dde",
+      "deletedDateTime": null,
+      "appId": "5b2c581d-e3ac-415e-a770-7f16254fdbf7",
+      "applicationTemplateId": null,
+      "disabledByMicrosoftStatus": null,
+      "createdDateTime": "2022-02-23T21:35:20Z",
+      "displayName": "active-directory-dotnet-minimal-api-aspnetcore-client-credentail-flow",
+      "description": null,
+      "groupMembershipClaims": null,
+      "identifierUris": [],
+      "isDeviceOnlyAuthSupported": null,
+      "isFallbackPublicClient": null,
+      "notes": null,
+      "publisherDomain": "contoso.onmicrosoft.com",
+      "serviceManagementReference": null,
+      "signInAudience": "AzureADMyOrg",
+      "tags": [],
+      "tokenEncryptionKeyId": null,
+      "defaultRedirectUri": null,
+      "certification": null,
+      "optionalClaims": null,
+      ...
+   }
+   ```
+
+## About the code
+
+This ASP.NET Core minimal web API has a single route (_/application_) that supports anonymous access.  When the anonymous route is called, the API requests its own application object from Microsoft Graph.
+
+The Microsoft Graph response data is included in its response back to the original caller. This web API uses [Microsoft Authentication Library (MSAL)](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet).
```

Comment:
Please disambiguate "its" and "original caller** in this sentence by stating exactly who is sending the response and exactly who the "original" caller is in this scenario:

> The Microsoft Graph response data is included in **its** response back to the **original caller**.

Reply:
```suggestion
The Microsoft Graph response data is included in console app response back to the client making the http request call (original caller). This web API uses [Microsoft Authentication Library (MSAL)](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet).
```

could you please assist me in here @ckittel with a second pass contrib to calibrate/normalize this a bit more? Im planning to propose this version, otherwise. 

cc/ @skabou

Reply:
How about?

```suggestion
The result from Microsoft Graph is returned as the response of this web API. This web API uses [Microsoft Authentication Library (MSAL)](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet).
```